databricks/koalas/tests/test_dataframe.py

```diff
@@ -1103,7 +1103,7 @@ def test_to_numpy(self):
 
     def test_to_pandas(self):
         pdf, kdf = self.df_pair
-        self.assert_eq(kdf.toPandas(), pdf)
+        self.assert_eq(kdf.to_pandas(), pdf)
```

Comment:
Shall we keep this one to check `toPandas` still works.

Reply:
Btw, shall we mark `toPandas()` as deprecated?

Reply:
Sounds good. Let me mark it in this PR.
Thanks for the suggestion! 👍